### PR TITLE
Allow (none) for CertificateValidationMethod

### PR DESCRIPTION
**Pull request by copelco**
Support option to manage SSL certificates outside of stack creation workflow.

### CODE REVIEW

File: stack/load_balancer.py [hunk 1]
```diff
@@ -106,14 +106,15 @@
...
+    from .certificates import application, cert_condition
+    listeners.append(If(cert_condition, elb.Listener(
+        Condition=cert_condition,
         LoadBalancerPort=443,
         InstanceProtocol=web_worker_protocol,
         InstancePort=web_worker_port,
         Protocol='HTTPS',
-        SSLCertificateId=application_certificate,
-    ))
+        SSLCertificateId=application,
```
Comment (dpoirier) [inline stack/load_balancer.py]: I think `application_certificate` is a better name, actually. Can we use that?  `application` doesn't suggest that it's a certificate.

File: stack/certificates.py [hunk 2]
```diff
@@ -12,20 +13,27 @@
...
+        AllowedValues=[dont_create_value, 'DNS', 'Email'],
         Type='String',
         Description=""
         "How to validate domain ownership for issuing an SSL certificate - "
-        "highly recommend DNS. Either way, stack creation will pause until "
-        "you do something to complete the validation."
+        "highly recommend DNS. DNS and Email will pause stack creation until "
+        "you do something to complete the validation. If omitted, an HTTPS "
+        "listener can be manually attached to the load balancer after stack "
+        "creation."
```
Comment (dpoirier) [inline stack/certificates.py]: Do we document how to do this? (I know we do in the sixfoot provisioning.rst document, just not sure about aws-web-stacks.)

Comment (copelco) [reply stack/certificates.py]: Added to README

**Comment by copelco** [comment]
Updated based on your feedback @dpoirier

**Pull request merged**